content/en/_index.md

```diff
@@ -0,0 +1,240 @@
+summary: Serverless Workflow presents a vendor-neutral, open-source, and entirely community-driven ecosystem tailored for defining and executing DSL-based workflows in the realm of Serverless technology.
+keywords:
+- workflows
+- serverless
+- cloudevent
+- services
+images:
+- img/logos/sw-logo.png
+---
+
+{{% blocks/cover title="Serverless Workflow" image_anchor="top" height="auto" %}}
+<a class="btn btn-lg btn-primary me-3 mb-4" href="https://github.com/serverlessworkflow/specification/blob/main/README.md">
+  Learn More <i class="fas fa-arrow-alt-circle-right ms-2"></i>
+</a>
+<p class="lead mt-5">Serverless Workflow presents a vendor-neutral, open-source, and entirely community-driven ecosystem tailored for defining and executing DSL-based workflows in the realm of Serverless technology.</p>
+
+{{< blocks/link-down color="info" >}}
+{{% /blocks/cover %}}
+
+
+{{% blocks/lead color="primary" %}}
+The Serverless Workflow DSL is a high-level language that reshapes the terrain of workflow creation, boasting a design that is ubiquitous, intuitive, imperative, and fluent.
+
+{{% /blocks/lead %}}
+
+
+{{% blocks/section color="dark" type="row" %}}
+{{% blocks/feature icon="fa-thumbs-up" title="Easy to Use!" %}}
+Serverless Workflow DSL simplifies workflow concepts, allowing users to easily create complex workflows.
+{{% /blocks/feature %}}
+
+
+{{% blocks/feature icon="fa-bolt" title="Event Driven" %}}
+Integrate events seamlessly into workflows with support for formats like CloudEvents, enabling event-driven architectures
+{{% /blocks/feature %}}
+
+
+{{% blocks/feature icon="fa-cogs" title="Service Oriented" %}}
+The Serverless Workflow DSL allows developers to integrate with service-oriented architectures, defining workflows that interact with services using protocols like HTTP, GRPC, OpenAPI, and AsyncAPI.
+{{% /blocks/feature %}}
+
+{{% /blocks/section %}}
+
+{{% blocks/section %}}
+Reach out to us!
+{.h1 .text-center}
+
+{{% /blocks/section %}}
+
+{{% blocks/section type="row" %}}
+
+{{% blocks/feature icon="fas fa-users" title="Join our Meetings!" url="https://github.com/serverlessworkflow/specification/discussions/799" %}}
+Add the schedule to your calendar and become a part of our discussions!
+{{% /blocks/feature %}}
+
+{{% blocks/feature icon="fab fa-github" title="Contributions welcome!"
+    url="https://github.com/serverlessworkflow/specification" %}}
+We do a [Pull Request](https://github.com/serverlessworkflow/specification/pulls)
+contributions workflow on **GitHub**. New users are always welcome!
+{{% /blocks/feature %}}
+
+{{% blocks/feature icon="fab fa-slack" title="Join us on Slack!"
+    url="https://slack.cncf.io/" %}}
+Chat with our community and follow announcements at **#serverless-workflow**
+
+{{% /blocks/feature %}}
+
+{{% /blocks/section %}}
+
+{{% blocks/section %}}
+
+Open Source projects supporting our DSL
+{.h1 .text-center}
+
+{{% /blocks/section %}}
+
+{{% blocks/section type="row" color="white" %}}
+
```

Comment:
We should always order aphabetically IMHO, so that we avoid opiniated - therefore discussable - ordering in this kind of cases 😉